test/server-unit/util.spec.js

```diff
@@ -0,0 +1,38 @@
+const util = require('../../server/lib/util');
+const { expect } = require('chai');
+var _ = require('lodash');
+
+const newLocal = function () {
```

Comment:
In the future, it's probably a good idea to call this function "UtilTests" or something like that.  That way the function name has meaning.